test/errors/handle.test.ts

```diff
@@ -71,6 +71,7 @@ describe('handle', () => {
   })
 
   fancy
+  .skip()
```

Comment:
`handle.ts` uses a promise here for `flush` https://github.com/oclif/core/blob/79965f3518ac9c0cc048ad3b3788946126cebc82/src/errors/handle.ts#L30-L31

but it's called from synchronous code.  

It would be better if `handle` were async.  I think this might work IRL since it's the last thing called, after errors, on commands, but it seems wrong.  That would be a breaking change, given how much code refers to `handle`.  

It *definitely* plays badly with the UT framework.